### ISSUE TITLE
Ability for shards to use a custom material

### ISSUE THREAD

**Issue by ZachAR3** [Jummit/godot-destruction-plugin#16]
I saw something like this mentioned in the commits but I don't see it available in the plugin, basically just a material slot which if found not null at time of shard generation will be used on all of the shards.

**Comment by Jummit** [comment]
This can be achieved by overriding the material in the shards scene. Is that an option for you?

**Comment by ZachAR3** [comment]
> This can be achieved by overriding the material in the shards scene. Is that an option for you?

Assuming you mean just manually editing the shard GLBS material; I mean that's a basic Godot feature, but that's kinda a pain compared to just having an exported attribute as part of your destructibles plugin haha. Obviously up to you though, I just think that's a worthwhile QOL update.

**Comment by Jummit** [comment]
Just to make sure that we are talking about the same thing: You can open the import dialog for the shard scene, go to the materials tab and set the "use external" to your material. Still, I can see this being inconvenient if you have to do it for a lot of scenes.

**Comment by ZachAR3** [comment]
> Just to make sure that we are talking about the same thing: You can open the import dialog for the shard scene, go to the materials tab and set the "use external" to your material. Still, I can see this being inconvenient if you have to do it for a lot of scenes.

Yup, that's I'm referring too. By allowing this too you could update materials for specific items or scenes instead of remaking the shard scene for multiple colors and whatnot.
